FireControl.cps

```diff
@@ -10,8 +10,8 @@
  FORKID {C59C057C-1427-4281-AE93-4F04BBA3F45E}
  */
 
-description = "FireControl Plasma v1.6";
-vendor = "Langmuir Systems";
+description = "FireControl Plasma v1.7";
```

Comment:
dont increment the minor version, instead use 1.6.1

Reply:
How can this not be categorized as a minor version increment?
https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning
This PR seems to satisfy the definition of "new, non-breaking feature" used in the article, as it is not just a minor bugfix that an increment to patch level would imply.